src/EditorConfigSamples.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can we use a .NET Core project? All this XML with these old style `csproj` files projects hurts my eyes.

Reply:
Feel free to convert or delete afterwards.